src/Components/Login/Login.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import style from './Login.module.scss'
+import aboutIcon from '../../assets/navbar/aboutNavIcon.png'
+import settingsIcon from '../../assets/navbar/settingsNavIcon.png'
+import statsIcon from '../../assets/navbar/statsNavIcon.png'
+import timerIcon from '../../assets/navbar/timerNavIcon.png'
+
+const Login = () => {
+  return (
+    <section className={style.loginContainer}>
+      <h2 className={style.loginHeader}>Welcome to Som Timer</h2>
+      <p className={style.appDescription}>
+        Som Timer is a timer that cares. This application serves curated wellness content on break intervals. Be mindful of rest so you can focus best!
+        <br /><br />
+        Sign in with Google to get started.
```

Comment:
I LOVE the layout of this walkthrough overall - totally on brand. I was just wondering what you would think of moving the Login button to below the tutorial, or if you specifically placed it above for a particular reason. My thought is that for a new user, we want them to skim the walkthrough before they log in, but I can also see how it would potentially be more convenient to have further up.

Reply:
I would agree with the placement of the login button being at the bottom... sort of taking the "gift shop before the exit" approach. Otherwise I think this is perfect!

Reply:
So, this works out except for in mobile view!! You'd have to scroll to login...

<img width="1438" alt="Screen Shot 2020-11-20 at 4 50 23 PM" src="https://user-images.githubusercontent.com/62263439/99860507-7d083500-2b50-11eb-9e3b-4fe25030c894.png">
<img width="633" alt="Screen Shot 2020-11-20 at 4 48 48 PM" src="https://user-images.githubusercontent.com/62263439/99860439-50541d80-2b50-11eb-8983-a60611387188.png">
<img width="374" alt="Screen Shot 2020-11-20 at 4 48 20 PM" src="https://user-images.githubusercontent.com/62263439/99860399-34507c00-2b50-11eb-8200-a7e2fcca3210.png">